schema/deploy/warehouse/site/data.sql

```diff
@@ -85,6 +85,7 @@ insert into warehouse.site(identifier, details)
         ('UWSuzzalloLibrary',                       '{"category": "community",  "type": "collegeCampus",
             "swab_site": "%suzzal+o"}'),
         ('UWReopeningSwabNSend',                    '{"category": "community",  "type": "uw-reopening"}'),
+        ('UWReopeningDropbox',                      '{"category": "community",  "type": "uw-reopening"}'),
```

Comment:
Does the lab record information about which specific dropbox the sample was collected from? There could be analytical benefits to distinguishing the sites, rather than lumping them all together, the same way we do for swab and send